spec/oncocore_vs.txt

```diff
@@ -94,25 +94,22 @@ SCT#250724005 "Tumor marker measurement (procedure)"
 SCT#386344002 "Laboratory data interpretation (procedure)"
 //MTH#C0005516	"Biological markers"
 
-ValueSet:		CancerStageTimingVS
+ValueSet:		TimeOfClassificationVS
 //TODO: the value set is missing codes for recurrence and anatomical staging.
-//TODO: need new terms for post-therapeutic clinical and pathological staging.
```

Comment:
We're still missing codes for posttherapy clinical and pathologic staging, so I would keep the comment. I know the VS is extensible, but we should treat these the same as recurrence and anatomical.

Reply:
Restored.